src/acp/internal/getDefaultAgentMatcherPolicyUrl.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2021 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import type { WithAccessibleAcr } from "../acp";
+import type { AccessModes } from "../type/AccessModes";
+import {
+  DefaultAccessControlName,
+  getDefaultAccessControlUrl,
+} from "./getDefaultAccessControlUrl";
+
+/** @hidden */
+export function getDefaultAgentMatcherPolicyUrl(
+  resource: WithAccessibleAcr,
+  name: DefaultAccessControlName,
+  mode: keyof AccessModes
+): string {
+  return getDefaultAccessControlUrl(resource, name)
```

Comment:
When the release is over, I would like to discuss this. I think it's fundamentally wrong to encode semantics in the IRI fragment, and for instance it's this type of practice that made Podbrowser hard to migrate to the latest ACP changes. I don't think we have a choice until the release, but I just wanted to raise this point for discussion later.